plugins/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/wizard/v3/NewOpenShift3ApplicationWizard.java

```diff
@@ -27,25 +29,45 @@
  * - proceed through wizard and set up missing remaining parameters if there are any
  * 
  * @author mlabuda@redhat.com
+ * @author adietish@redhat.com
  */
-public class NewOpenShift3ApplicationWizard extends NewOpenShiftApplicationWizard {
+public class NewOpenShift3ApplicationWizard extends AbstractOpenShiftApplicationWizard {
 
 	public NewOpenShift3ApplicationWizard() {
 		super(DatastoreOS3.SERVER, DatastoreOS3.USERNAME);
```

Comment:
probably this is the place where constructor with a project parameter should be used and in open methods could be than used the project value?